patches_for_WebRTC_org/m84/src/modules/video_capture/windows/video_capture_winrt.cc

```diff
@@ -513,15 +513,11 @@ HRESULT VideoCaptureWinRTInternal::FrameArrived(
       hr = bitmap_buffer->GetPlaneCount(&plane_count);
     }
 
-    if (SUCCEEDED(hr)) {
-      hr = plane_count == 2 ? S_OK : E_FAIL;
-    }
-
-    if (SUCCEEDED(hr)) {
+    if (SUCCEEDED(hr) && plane_count >= 1) {
```

Comment:
This change treats 1 plane and 2 plane frames identically. Is that intentional or is there any different work that needs to be done in the 1 plane case?

Reply:
It's intentional: `pfn_incoming_frame_` is able to figure out the data format and to decode it correctly.